app/views/games/show.html.haml

```diff
@@ -109,14 +109,24 @@
       - prevscore = -1
       - currentnum = 1
       - countnum = 1
-      - @game.registrations.sort_by{|x| x.display_score}.reverse_each do |p|
+      all_players = @game.registrations.group_by(:faction_id).sort_by{|x| x.display_score}.reverse_each do |p|.flatten
+        @humans = p
```

Comment:
I'm really confused about this `@humans = p` and the `@zombies = p` below. They don't seem to do anything